src/components/bulk-email-tool/bulk-email-form/BulkEmailForm.jsx

```diff
@@ -68,12 +69,14 @@ function BulkEmailForm(props) {
    * @returns formatted Data
    */
   const formatDataForFormAction = (action) => {
+    const emailsFormat = emailLearnersList.map(({ email }) => email);
     if (action === FORM_ACTIONS.POST) {
       const emailData = new FormData();
       emailData.append('action', 'send');
       emailData.append('send_to', JSON.stringify(editor.emailRecipients));
       emailData.append('subject', editor.emailSubject);
       emailData.append('message', editor.emailBody);
+      emailData.append('email_learners', JSON.stringify(emailsFormat));
```

Comment:
can we use: `individual-students`?

Reply:
or if learners are used across the app: individual-learners-emails

Reply:
I can use it with underscore individual_learners_emails